lib/ProductOpener/Images.pm

```diff
@@ -1314,7 +1314,7 @@ sub display_image($$$) {
 
 		my $path = product_path($product_ref);
 		my $rev = $product_ref->{images}{$id}{rev};
-		my $alt = remove_tags_and_quote($product_ref->{product_name}) . ' - ' . $Lang{$imagetype . '_alt'}{$lang};
+		my $alt = remove_tags_and_quote($product_ref->{product_name}) . ' - ' . $Lang{$imagetype . '_alt'}{$lang} . ' - ' . $display_lc;
```

Comment:
So $display_lc is the language of the interface. e.g. on https://in.openfoodfacts.org, $display_lc is "en" (English). On https://fr.openfoodfacts.org , it is "fr".

But if we don't have an image in $display_lc, we will display an image for another language.

See this code:

# first try the requested language
	my @display_ids = ($imagetype . "_" . $display_lc);		my @display_ids = ($imagetype . "_" . $display_lc);
	# next try the main language of the product		# next try the main language of the product
	if ($product_ref->{lc} ne $display_lc) {		if ($product_ref->{lc} ne $display_lc) {
		push @display_ids, $imagetype . "_" . $product_ref->{lc};			push @display_ids, $imagetype . "_" . $product_ref->{lc};
	}		}

To solve this bug, we want to display in the alt tag the language of the actual image displayed, not the language of the interface.

Reply:
When I did not added image in $display_lc, while image was added for the other language, no ingredient image was displayed.

Reply:
@stephanegigandet When I did not add image in $display_lc, while images were added for other languages, no ingredient image was displayed.

Reply:
@stephanegigandet kindly review